src/test/ui/range/issue-54505-no-literals.fixed

```diff
@@ -0,0 +1,85 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
```

Comment:
Copyright notices are no longer needed — you can just get rid of this (and elsewhere).